modules/vf-graphql-holochain/resolvers/resourceSpecification.ts

```diff
@@ -46,10 +46,11 @@ export default (enabledVFModules: VfModule[] = DEFAULT_VF_MODULES, dnaConfig: DN
       },
     } : {}),
     (hasMeasurement ? {
-      defaultUnitOfResource: () => {
-        // issue #155
-        // https://github.com/h-REA/hREA/issues/155
-        throw new Error('resolver unimplemented')
+      defaultUnitOfResource: async (record: { defaultUnitOfResource: AddressableIdentifier }) => {
+        if (!record.defaultUnitOfResource) {
```

Comment:
can use the destructured property here, no?

Reply:
would have to destructure the left side of this
`record: { defaultUnitOfResource: AddressableIdentifier }`

Reply:
*shrugs* it's a minor thing :)